observability/otel/otel-forwarding-logs.adoc

```diff
@@ -0,0 +1,172 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="otel-forwarding-logs"]
+= Forwarding Logs to a LokiStack instance
+include::_attributes/common-attributes.adoc[]
+:context: otel-forwarding-logs
+
+To configure forwarding logs to a LokiStack instance, you can deploy and configure the OpenTelemetry Collector. You can deploy the OpenTelemetry Collector in the deployment mode by using the specified processors, receivers, and exporters.
+
+.Prerequisites
+
+* The {OTELOperator} is installed.
+* The {loki-op} is installed. See _Additional resources_ on how to install LokiStack.
+* A supported LokiStack instance is deployed on the cluster.
+
+TODO add dev-preview note and that the `lokipush` exporter will be removed in future release in favour of OTLPHTTP.
+
+.Procedure
+
+. Create a service account for the OpenTelemetry Collector.
++
+.Example ServiceAccount
+[source,yaml]
+----
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: otel-collector-deployment
```

Comment:
missing namespace, add:
```
namespace: openshift-logging
```

Reply:
Thank you. I've added it in my PR.
@IshwarKanse, FYI ^.